### ISSUE TITLE
react: Apostrophes in strings are rendered incorrectly

### ISSUE THREAD

**Issue by mfulton26** [storybookjs/storybook#9589]
**Describe the bug**
Apostrophes are rendered as `&#39;` instead of as `'`.

**To Reproduce**
1. Go to https://storybooks-official.netlify.com/?path=/story/addons-knobs-withknobs--tweaks-static-values
2. Click on **Knobs**
3. Change **Name** from "Storyteller" to "Mc'Storyteller"
4. See error rendered on the **Canvas**: `My name is Mc&#39;Storyteller`

**Expected behavior**
Canvas should be rendered with `My name is Mc'Storyteller`

**Code snippets**
Any `string` used in JSX containing an apostrophe will be rendered incorrectly.

e.g.

```jsx
render() {
  const string = "I'm wrong";
  return <>{string}</>;
}
```

**System:**

```
  System:
    OS: macOS Mojave 10.14.6
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
  Binaries:
    Node: 12.14.0 - /usr/local/bin/node
    Yarn: 1.21.1 - /usr/local/bin/yarn
    npm: 6.13.4 - /usr/local/bin/npm
  Browsers:
    Chrome: 79.0.3945.117
    Firefox: 70.0.1
    Safari: 13.0.4
  npmPackages:
    @storybook/addon-actions: ^5.2.8 => 5.2.8 
    @storybook/addon-knobs: ^5.2.8 => 5.2.8 
    @storybook/react: ^5.2.8 => 5.2.8 
```

**Additional context**

Apostrophes in JSX text nodes will render correctly (from what I've seen so far):

```jsx
render() {
  return <>I'm right</>;
}
```

So this issue might only apply to strings that are used within JSX elements (and not to JSX text nodes directly).

**Comment by joey80** [comment]
I'm experiencing this issue with Vue as well

**Comment by cellog** [comment]
This is still happening, is there any reason it's been ignored or just inertia?

**Comment by shilman** [comment]
@cellog we’ve released `addon-controls` in Storybook 6.0. Controls are portable, auto-generated knobs that are intended to replace `addon-knobs` long term.

- [Controls documentation](https://storybook.js.org/docs/react/essentials/controls)
- [Migration instructions](https://github.com/storybookjs/storybook/blob/next/addons/controls/README.md#how-do-i-migrate-from-addon-knobs)

PRs on addon-knobs still welcomed
